nifi-h2/nifi-h2-database-migrator/src/main/java/org/apache/nifi/h2/database/migration/H2DatabaseMigrator.java

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.nifi.h2.database.migration;
 
-import org.apache.nifi.org.h2.jdbcx.JdbcDataSource;
+import org.h2.jdbcx.JdbcDataSource;
```

Comment:
The most recent GitHub builds failed due to this migrator change, which should be reverted so that it continues to reference the shaded class.